Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonButton.cs

```diff
@@ -228,6 +227,18 @@ private void ResetButtonStyle()
             ButtonStyle = ButtonStyle.Standalone;
         }
 
+        [DefaultValue(true), Description("Always use the set text, even if the 'DialogResult' value has been changed. (Note: You can manage the 'DialogResult' string values via the KryptonManager on the parent KryptonForm.)")]
```

Comment:
When this class is initialised.. What is setting this value to true ?
The `DefaultValue` is just for the designer, Not the initial field value.